components/eamxx/docs/mkdocs/docs/user/index.md

```diff
@@ -0,0 +1,2 @@
+# SCREAM User Guide
```

Comment:
Maybe we should get rid of this, because E3SM's User Guide strategy is all-Confluence, all the time (which I think is fine).

Reply:
You mean completely remove the user guide part?